tests/python/direct/README.md

```diff
@@ -65,20 +65,16 @@ The following tests only exist in legacy frontend:
 - `test_fusion_profiler_user_schedule` - Tests user-defined fusion profiling; Cuda 13 incompatibility
 - `test_fusion_profiler_with_noncodegen_kernels` - Tests profiling with non-codegen kernels; Cuda 13 incompatibility
 - `test_cuda_code_and_scheduled_fusion_ir_strings` - Tests CUDA code generation (101 lines)
+- `test_arithmetic_ops` - TODO: Tests __neg__ and __abs__ arithmetic operations
```

Comment:
@jjsjann123 I would look at `tests/python/direct/README.md` until the end of the stack at #5131.